Doberman/Node.py

```diff
@@ -230,36 +273,68 @@ def process(self, packages):
 class PolynomialNode(Node):
     """
     Does a polynomial transformation on a value
+
+    Setup params:
+    None
+
+    Runtime params:
+    :param transform: list of numbers, the little-endian-ordered coefficients. The
+        calculation is done as a*v**i for i,a in enumerate(transform), so to output a 
+        constant you would specity [value], to leave the input unchanged you would
+        specify [0, 1], a quadratic could be [c, b, a], etc
     """
     def process(self, package):
         xform = self.config.get('transform', [0,1])
         return sum(a*package[self.input_var]**i for i,a in enumerate(xform))
 
 class InfluxSinkNode(Node):
     """
-    Puts a value back into influx
+    Puts a value back into influx. The sensor tag will be "pipeline"
+
+    Setup params:
+    :param topic: string, the type of measurement (temperature, pressure, etc)
+    :param subsystem: string, optional. The subsystem the quantity belongs to
+
+    Runtime params:
+    None
     """
     def setup(self, **kwargs):
         super().setup(**kwargs)
         self.topic = kwargs['topic']
+        self.subsystem = kwargs.get('subsystem')
         self.write_to_influx = kwargs['write_to_influx']
 
     def process(self, package):
         if not self.is_silent:
-            self.write_to_influx(topic=self.topic, tags={'reading': self.output_var, 'sensor': 'pipeline'},
-                                fields={'value': package[self.input_var]}, timestamp=package['time'])
+            self.write_to_influx(topic=self.topic, tags={'reading': self.output_var,
+                'sensor': 'pipeline', 'subsystem': self.subsystem},
+                fields={'value': package[self.input_var]}, timestamp=package['time'])
 
 class EvalNode(Node):
     """
-    An evil node that executes an arbitrary operation specified by the user
+    An evil node that executes an arbitrary operation specified by the user.
```

Comment:
Oh no! why ist the note so evil? :smiling_imp:

Reply:
Because running `eval` on unsanitized user input is a dangerous game. I don't like that we have this but I also don't want to spend the time to develop something safer.

Reply:
E.g. maybe useful https://numexpr.readthedocs.io/projects/NumExpr3/en/latest/